packages/neutrino-preset-karma/package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "neutrino-preset-karma",
-  "version": "4.2.0",
+  "version": "5.0.0",
   "description": "Neutrino preset for testing Neutrino projects with Karma",
-  "main": "src/index.js",
+  "main": "index.js",
```

Comment:
Just to flatten the directory structure for packages that don't need the extra complexity.